### DISCUSSION TITLE
articles/zen-baggage

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#173] [Comments]
# articles/zen-baggage

花了几天时间，读完了比尔·波特写的《禅的行囊》，这是我读过的他的第二本书。自从看了《心经解读》，我就对比尔·波特这个老头产生了兴趣，他写的书很符合我的口味。在《禅的行囊》中，作者从北京到香港，从北到南，穿行了小半个中国，寻访禅宗在中国的印记。既有对禅宗历史的追溯，也有对禅宗在中国当前的状态观察。

https://hutusi.com/articles/zen-baggage

**Comment by iamgodot** [comment]
禅宗好像很喜欢打机锋，也许是觉得语言难以交流更高的智慧。相比之下，西方哲学做了很多务实的逻辑推导，但最后可能还是会归于一个无字？

**Reply by hutusi** [reply]
玄而又玄，谓之众妙之门。哈哈。

从对人类知识的贡献角度出发，应该还是西方哲学更实用些。
